src/authorization/AuthorizationsDefinition.ts

```diff
@@ -532,12 +532,37 @@ const AUTHORIZATION_DEFINITION: AuthorizationDefinition = {
           args: { filters: ['OwnUser'] }
         },
         attributes: [
-          'id', 'userID', 'issuer', 'active', 'description', 'visualID', 'default',
+          'userID', 'issuer', 'active', 'description', 'visualID', 'default',
           'user.id', 'user.name', 'user.firstName', 'user.email', 'user.issuer'
         ],
       },
       {
-        resource: Entity.TAG, action: [Action.DELETE, Action.UPDATE],
+        resource: Entity.TAGS, action: Action.UNASSIGN,
+        condition: {
+          Fn: 'custom:dynamicAuthorizations',
+          args: { filters: ['OwnUser'] }
```

Comment:
I should have 'assert' here to ensure that the Basic user can only change his own badge.